Sources/CreateOptions/GenerateOptions+Acronyms.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+// TODO: This needs making thread safe.
+extension GenerateOptions {
+    private static let defaultAcronyms: Set<String> = ["url", "id", "html", "ssl", "tls", "https", "http", "dns", "ftp", "api", "uuid", "json"]
+    private static var cache: [Int: [String]] = [:]
+
+    /// A set of all acronyms based on the default list after factoring in the `addedAcronyms` and removing `ignoredAcronyms`.
+    /// Results are ordered so that the longer acronyms come first.
+    public var allAcronyms: [String] {
+        var hasher = Hasher()
+        hasher.combine(addedAcronyms)
+        hasher.combine(ignoredAcronyms)
+        let hashValue = hasher.finalize()
+
+        if let cached = Self.cache[hashValue] {
+            return cached
+        }
+
+        let allAcronyms = Self.defaultAcronyms
+            .union(addedAcronyms)
+            .subtracting(ignoredAcronyms)
+            .sorted { $0.count > $1.count }
+
+        Self.cache[hashValue] = allAcronyms
+        return allAcronyms
+    }
+}
```

Comment:
I took this approach so that `GenerateOptions` could remain a struct, but now that I am thinking that it would be better to just keep `GenerateOptions` as a class instead. 

We have to be careful because we want to only compute this value once, but its accessed across multiple threads so this would have likely been a problem... I'll think of something better..